src/Components/Publishing/Layouts/ArticleWithFullScreen.tsx

```diff
@@ -78,12 +78,22 @@ export class ArticleWithFullScreen extends React.Component<
 
   render() {
     const { article, fullscreenImages } = this.state
-    const { closeViewer, marginTop, slideIndex, viewerIsOpen } = this.props
+    const {
+      closeViewer,
+      marginTop,
+      slideIndex,
+      viewerIsOpen,
+      onOpenAuthModal,
+    } = this.props
 
     const articleProps = extend(cloneDeep(this.props), { article, slideIndex })
 
     return (
-      <TooltipsData article={article} shouldFetchData={this.props.showTooltips}>
+      <TooltipsData
+        article={article}
+        shouldFetchData={this.props.showTooltips}
+        onOpenAuthModal={onOpenAuthModal}
```

Comment:
I like how you've set tooltips up to use context, now seeing that a lot of our info for this feature is consolidating there, we should probably use this for the showTooltips props too 🤔 not for this pr just noting